commands/model/get/mod.ts

```diff
@@ -0,0 +1,60 @@
+import { asserts } from "../../../lib/asserts.ts";
+import { loading } from "../../../lib/loading.ts";
+import { input } from "../../../prompts/input.ts";
+import { args, command, flags, z } from "../../../zcli.ts";
+import { dataTable } from "../../../lib/data-table.ts";
+import { fields } from "../../../flags.ts";
+import { pickJson } from "../../../lib/pick-json.ts";
+import { models } from "../../../api/models.ts";
+
+/**
+ * This variable is automatically generated by `zcli add`. Do not remove this
+ * or change its name unless you're no longer using `zcli add`.
+ */
+const subCommands: ReturnType<typeof command>[] = [];
+
+export const get = command("get", {
+  short: "Get a model",
+  long: `
+    Get a model from a team.
+  `,
+  commands: subCommands,
+  args: args().tuple([
+    z.string().describe("The ID of the model to get"),
+  ]).optional(),
+  flags: flags({
+    fields,
+  }),
+  // We use command metadata in the `persistentPreRun` function to check if a
+  // command requires an API key. If it does, we'll check to see if one is
+  // set. If not, we'll throw an error.
+  meta: {
+    requireApiKey: true,
+  },
+}).run(
+  async function* ({ args, flags }) {
+    let [id] = args;
+
+    if (!id) {
```

Comment:
When the user doesn't have an ID on hand, it's useful to fall back to selection e.g. https://github.com/Paperspace/cli/blob/main/commands/deployment/delete/mod.ts#L36-L55